src/App.vue

```diff
@@ -14,20 +14,14 @@
             src="./assets/beacon-network-logo.png"
         /></router-link>
       </div>
-      <a
-        v-if="!getCookie('logged_in')"
-        class="login"
-        href="https://auth-beacon.rahtiapp.fi/login"
+      <a v-if="!getCookie('logged_in')" class="login" :href="login_url"
         ><img src="./assets/elixir-login.png"
       /></a>
-      <b-button
-        v-if="getCookie('logged_in')"
-        class="login"
-        v-on:click="logOut"
-        type="is-primary"
-        size="is-medium"
-        >Log Out</b-button
-      >
+      <a v-if="getCookie('logged_in')" class="login" :href="logout_url">
+        <b-button class="login" type="is-primary" size="is-medium"
```

Comment:
would be nice to fix this design of the `Logout` button to make it a bit more consistent than:
![image](https://user-images.githubusercontent.com/47524/67617246-e270e180-f7e9-11e9-8531-5eabe8a5aafc.png)

A suggestion for improvement would be:
1. remove the logged in or add a the elixir user id/user name instead 
   * if decision is to keep it probably make the log in as a text instead of a `tag`
2. make the `log out` button of default size by removing the `size="is-medium"`

Reply:
Done